app/src/main/java/ch/epfl/sweng/eventmanager/ui/eventShowcase/fragments/EventMainFragment.java

```diff
@@ -34,6 +36,15 @@ public void onResume() {
 
             TextView eventDescription = (TextView) view.findViewById(R.id.event_description);
             eventDescription.setText(ev.getDescription());
+            Button contactButton = view.findViewById(R.id.contact_form_go_button);
```

Comment:
We may want to put this in the lateral menu

Reply:
I think for now I would put it here, this can be easely changed in the future
Anyway tests do not pass

Reply:
@zyuiop if you want the button to be displayed elsewhere please create an issue so that I can merge this one and keep thing simple :) 
It is currently only a way of accessing the form and certainly is not its final place !